tests/test_normalized_gym.py

```diff
@@ -39,9 +35,8 @@ def test_unflatten(self):
             for e in range(5):
                 action = env.action_space.sample()
                 next_obs, reward, done, info = env.step(action)
-                assert flatten(env.observation_space,
-                               next_obs).shape == flat_dim(
-                                   env.observation_space)
+                assert np.asarray(next_obs).flatten(
```

Comment:
This is a logical error.

a.flatten().shape == b.shape iff a is already flat.

Reply:
would would be the correct logic here? a.shape == b.shape?

Reply:
This should be something like this:
`observation_space.flatten(next_obs).shape[0] == observation.flat_dim`
Of course the this next_obs should not be a batch of observation.